docs/reference/models/index.rst

```diff
@@ -0,0 +1,15 @@
+Models
+======
+
+Craft application uses `Pydantic`_ to describe the models used in ``<app>.yaml`` files.
+This helps generate `JSON schemas`_ for the files, improving editor integration.
```

Comment:
```suggestion
This helps generate `JSON schemas`_ for the files, which facilitate text editor and IDE integration.
```

Reply:
I've changed this to "which facilitate editor integration" (which covers both text editors and IDEs)